### PR TITLE
View/Module for admin page

### PR DESCRIPTION
**Pull request by ykostov**
WIP

### CODE REVIEW

File: lib/bgsite_official_web/views/home_view.ex [hunk 1]
```diff
@@ -0,0 +1,3 @@
+defmodule BgsiteOfficialWeb.AdminView do
+  use BgsiteOfficialWeb, :view
+end
```
Comment (ykostov) [inline lib/bgsite_official_web/views/home_view.ex:3]: This file must be `admin_view.ex`

File: lib/bgsite_official_web/templates/admin_controller/index.html.eex [hunk 2]
```diff
@@ -0,0 +1 @@
+<h1> Hello </h1>
```
Comment (ykostov) [inline lib/bgsite_official_web/templates/admin_controller/index.html.eex:1]: The templates directory is only for templates :), remove admin_controller inside templates (and admin_controller/index)

File: lib/bgsite_official_web/templates/admin/index.html.eex [hunk 3]
```diff
@@ -0,0 +1 @@
+<%= def render(index.html)%>
```
Comment (ykostov) [inline lib/bgsite_official_web/templates/admin/index.html.eex:1]: Thats is our html, no need for dynamic elm. yet, just put a `<h1> Testing Admin Template </h1>`

File: lib/bgsite_official_web/controllers/admin_controller.ex [hunk 4]
```diff
@@ -0,0 +1,9 @@
+defmodule  BgsiteOfficialWeb.AdminController do
+  use BgsiteOfficialWeb, :controller
+
+  def index(conn, _params) do
+    render(conn, "index.html")
+
+  end
+
+end
```
Comment (ykostov) [inline lib/bgsite_official_web/controllers/admin_controller.ex:9]: :+1:

File: lib/bgsite_official_web/templates/admin_controller/index.html.eex (hunk 2 above)
Comment (rmilushev) [reply lib/bgsite_official_web/templates/admin_controller/index.html.eex:1]: I agree with @ykostov :+1:

File: lib/bgsite_official_web/templates/admin/index.html.eex (hunk 3 above)
Comment (rmilushev) [reply lib/bgsite_official_web/templates/admin/index.html.eex:1]: I think also that ```index.html``` have to be in double quotes.

**Pull request merged**